### PR TITLE
Add check for Mac OS

### PR DESCRIPTION
**Pull request by ravsa**

### CODE REVIEW

File: hack/generate-db-migrations.sh [hunk 1]
```diff
@@ -37,8 +37,14 @@ done
...
-#for MAC docker run -t -v `pwd`:/bayesian \
-docker run -t -v "$(readlink -f "${THISDIR}/.."):/f8a_worker:z" \
+if [ "$(uname)" == "Darwin" ]; then
+    #for MAC docker
+    vol_mount="`pwd`:/f8a_worker:z"
+else
+    vol_mount="$(readlink -f "${THISDIR}/.."):/f8a_worker:z"
+fi
+
+docker run -t -v $vol_mount \
```
Comment (msrb) [inline hack/generate-db-migrations.sh]: I _think_ this will break for volume paths containing spaces.

Comment (msrb) [reply hack/generate-db-migrations.sh]: I.e. you want to retain the `" "`

Comment (ravsa) [reply hack/generate-db-migrations.sh]: Nice catch, Thanks

**Comment by ravsa** [comment]
[test]

File: hack/generate-db-migrations.sh (hunk 1 above)
Comment (msrb) [reply hack/generate-db-migrations.sh]: I meant doing something like this:
`docker run -t -v "$vol_mount"`

:)

**Pull request merged**